src/librawspeed/common/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ FILE(GLOB SOURCES
   "RawImageDataFloat.cpp"
   "RawImageDataU16.cpp"
   "RawspeedException.h"
+  "Spline.h"
+  "Spline.cpp"
```

Comment:
Yes, i *really* did mean `Spline.h`, **not** `Spline.cpp`

Reply:
Oh, do I not need to include the .cpp file in the list?

Reply:
I.e. there should not be `Spline.cpp`, just `Spline.h`.